### DISCUSSION TITLE
Welcome to yabai Discussions!

### DISCUSSION THREAD

**Discussion by asmvik** [asmvik/yabai#757] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 24unix** [comment]
Thanks for the invite.

I'm Micha, 50 years old self employed coder & admin living in Germany.

**Comment by dominiklohmann** [comment]
Hey everyone; I've started migrating some issues labeled `question` over to discussions. Feel free to discuss freely here or ask questions. I'll try to moderate this as I can fit into my current work schedule.

**Comment by rockyzhang24** [comment]
Thank you guys! @dominiklohmann is doing a really great job of migration 👍
